pom.xml

```diff
@@ -465,6 +466,16 @@
 					</excludes>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-checkstyle-plugin</artifactId>
+				<version>${checkstyle.version}</version>
+				<configuration>
+					<failOnViolation>false</failOnViolation>
+					<configLocation>configuration/checkstyle.xml</configLocation>
+					<suppressionsLocation>configuration/checkstyle-suppressions.xml</suppressionsLocation>
```

Comment:
Can we put this in ${builddir}? Otherwise we would have to exclude them from the built jar.